lib/features/identity_creator/presentation/identity_creator_view.dart

```diff
@@ -549,6 +549,7 @@ class IdentityCreatorView extends GetWidget<IdentityCreatorController>
     return HtmlEditor(
       key: controller.htmlKey,
       minHeight: controller.htmlEditorMinHeight,
+      adjustHeight: true,
```

Comment:
`adjustHeight: true` is the default value. So we don't need to add it.

Reply:
Done